constellations/gemini-admin/src/ask_for_help/afhEdit.js

```diff
@@ -81,38 +82,38 @@ export class AskForHelpEditPage extends React.Component {
                 .then((res) => successCallback())
                 .catch((err) => failCallback(err));
         }
-    }
+    };
 
     onClickCancel = () => {
         window.location.hash = "afh";
-    }
+    };
 
     onClickDelete = () => {
         this.setState({ showDeleteModal: true });
-    }
+    };
 
     onConfirmDelete = () => {
         const afhId = this.state.id;
         API.delete("api/askforhelp/afh/" + afhId).then((res) => {
             window.location.hash = "afh";
         });
-    }
+    };
 
     onSavedOk = () => {
         this.onDismissModal();
         window.location.hash = "afh";
-    }
+    };
 
     onDismissModal = () => {
         this.setState({
             showDeleteModal: false,
             showSaveModal: false,
         });
-    }
+    };
 
     onMomentChange = (newMoment) => {
         this.setState({ inputPostedAt: newMoment, date: newMoment });
-    }
+    };
```

Comment:
These semicolons aren't required in this case because we're defining functions here. In Javascript, I think they're not considered statements so we shouldn't need them :)

Reply:
Prettify puts the semicolons there, circleci fails without them - should I put them back or just leave it without them?

Reply:
oh hmmm... I'll look into this. Well let prettify keep them for now then